vuepress/docs/v7.3/docs/releases/README.md

```diff
@@ -4,98 +4,100 @@ sidebarDepth: 2
 
 # Entando 7.3 Release Notes
 
-Entando 7.3 is an important release with major advances and performance improvements, especially for the App Builder, content management UX and Multitenancy features.
+Entando 7.3 is an essential feature release with significant advances for Multitenancy and design and performance upgrades for the App Builder.
 
 ## Summary
-- [Multitenancy 2.0](#multitenancy) adds independent bundle functionality and access to registries for each tenant
-- App Builder introduces RBAC distributed editorial system 
-- Significant improvements to the performance and UX of the App Builder and its CMS
-- Additional performance, security, and dependency improvements
+- [Multitenancy 2.0](#multitenancy) adds full bundle functionality and access to registries independently across tenants.
+- Cloud ARM systems and ARM64-based architecture is now supported, including MacBooks with an M1 or M2 chip.
+- App Builder introduces a distributed editorial system with RBAC controls and page tree redesign.
+- New bundle features were added to automate the upgrade process to docker-based bundles and request basic resources for microservices. The bundle uninstall process was also improved with internal references to minimize technical debt. 
+- APISIX is now supported on Entando, including authentication, authorization, rate-limiting, and improved security features out of the box.
+- A mediator library has been added to improve communications between micro frontends in the portal UI via a broadcast event.
+- Additional Performance, security, and dependency improvements
 
 ### Compatibility
 * [Entando 7.3 Compatibility](https://entando.com/entando-de-app/cmsresources/cms/documents/Entando_7.3_Compatibility.pdf)
 * [Guides for previous versions of Entando](https://entando.com/page/en/compatibility-guide)
 
 ## Breaking Changes
-
+- 
```

Comment:
Am unsure what should be put under the Breaking changes, it seems like there isn't any clear breaking changes